integration/raft/migration_test.go

```diff
@@ -34,10 +35,9 @@ import (
 
 var _ = Describe("Kafka2RaftMigration", func() {
 	var (
-		testDir   string
-		client    *docker.Client
-		network   *nwo.Network
-		chaincode nwo.Chaincode
+		testDir string
+		client  *docker.Client
+		network *nwo.Network
 
 		process                ifrit.Process
 		peerProc, brokerProc   ifrit.Process
```

Comment:
One of the primary goals in using the broadcast client directly is to eliminate the need for peers from this test, so I would expect peerProc to go away.